### PR TITLE
CLDR-17081 Fix PNF test data for ICU

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-17081

This is not yet complete; it fixes some of the "inclusion of draft data" plus the stray superscript

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by macchiati** [comment]
Still draft for now, so just fyi

File: tools/cldr-code/src/main/java/org/unicode/cldr/tool/GeneratePersonNameTestData.java [hunk 1]
```diff
@@ -69,7 +71,9 @@ public static void main(String[] args) {
             }
 
             try {
-                CLDRFile cldrFile = factory.make(locale, true);
+                CLDRFile cldrFile =
+                        factory.make(locale, true, DraftStatus.contributed); // don't include
+                // draft=unconfirmed/provisional
```
Comment (richgillam) [inline tools/cldr-code/src/main/java/org/unicode/cldr/tool/GeneratePersonNameTestData.java:76]: I don't know what I'm looking at, but is `DraftStatus.contributed` a property of the whole file?  What I was seeing in some files was that _some_ person-name resources had their draft status set to "contributed" (or, actually, not mentioned at all), and _others_ had their draft status set to "provisional" or "unconfirmed".  Does this catch that?

**Review by richgillam — COMMENTED** [review]
Changes look good so far...

File: tools/cldr-code/src/main/java/org/unicode/cldr/tool/GeneratePersonNameTestData.java (hunk 1 above)
Comment (macchiati) [reply tools/cldr-code/src/main/java/org/unicode/cldr/tool/GeneratePersonNameTestData.java:92]: It sets the minimum bar for items to be included. So contributed means either contributed or approved (which is implicit if there is no draft value). That's the general bar we use for ICU, so this should get better results. cc @pedberg-icu

File: common/main/gl.xml [hunk 2]
```diff
@@ -9797,27 +9797,26 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
...
 		</personName>
 		<personName order="sorting" length="long" usage="referring" formality="informal">
 			<namePattern>{surname-core} {surname2}, {given-informal} {surname-prefix}</namePattern>
-			<namePattern alt="1">↑↑↑</namePattern>
+			<namePattern alt="1">{surname-core}, {given-informal} {surname-prefix}</namePattern>
 		</personName>
 		<personName order="sorting" length="medium" usage="referring" formality="formal">
 			<namePattern>{surname-core} {surname2}, {given} {given2-initial} {surname-prefix}</namePattern>
-			<namePattern alt="1">{surname-core} {surname2}, {given} {given2} {surname-prefix}</namePattern>
+			<namePattern alt="1">{surname-core}, {given} {given2} {surname-prefix}</namePattern>
```
Comment (richgillam) [inline common/main/gl.xml:9807]: Did you want this to be `{given2}` when the first entry has `{given2-initial}`?

**Review by richgillam — DISMISSED** [review]
All the changes look great.  Thank you!

- Is it worth looking through all the files to make sure we don't have any other `<namePattern alt="1">↑↑↑</namePattern>` stuff in there?
- Are there other changes coming, or is this all ready to go?

**Review by richgillam — APPROVED** [review]
Looks good!  But see question about `{given2-initial}` in my comments above.

File: common/main/gl.xml (hunk 2 above)
Comment (macchiati) [reply common/main/gl.xml:9807]: Missed this, will examine in 1/2 hour

Comment (macchiati) [reply common/main/gl.xml]: Got it (lined up all the others for comparison, too, so I wouldn't miss any others). Good catch.

**Review by richgillam — DISMISSED** [review]
Cut and print!

**Comment by macchiati** [comment]
Dang, forgot to rerun the GenerateTestData! Can you stamp again?

**Review by richgillam — APPROVED** [review]
Stamped again! 😀

**Pull request merged**